tests/backends/lrs/test_es.py

```diff
@@ -0,0 +1,386 @@
+                                    "verb.id.keyword": (
+                                        "http://adlnet.gov/expapi/verbs/attended"
+                                    )
+                                }
+                            },
+                            {"term": {"object.objectType.keyword": "Activity"}},
+                            {
+                                "term": {
+                                    "object.id.keyword": (
+                                        "http://www.example.com/meetings/34534"
+                                    )
+                                }
+                            },
+                        ]
+                    }
+                },
+                "sort": [{"timestamp": {"order": "desc"}}],
+                "track_total_hits": False,
+            },
+        ),
+        # 7. Query by timerange (with since/until).
+        (
+            {
+                "since": "2021-06-24T00:00:20.194929+00:00",
+                "until": "2023-06-24T00:00:20.194929+00:00",
+            },
+            {
+                "pit": {"id": "foo_pit_id", "keep_alive": "1m"},
+                "query": {
+                    "bool": {
+                        "filter": [
+                            {
+                                "range": {
+                                    "timestamp": {
+                                        "gt": datetime.fromisoformat(
+                                            "2021-06-24T00:00:20.194929+00:00"
+                                        )
+                                    }
+                                }
+                            },
+                            {
+                                "range": {
+                                    "timestamp": {
+                                        "lte": datetime.fromisoformat(
+                                            "2023-06-24T00:00:20.194929+00:00"
+                                        )
+                                    }
+                                }
+                            },
+                        ]
+                    }
+                },
+                "sort": [{"timestamp": {"order": "desc"}}],
+                "track_total_hits": False,
+            },
+        ),
+        # 8. Query with pagination and pit_id.
+        (
+            {"search_after": "1686557542970|0", "pit_id": "46ToAwMDaWR5BXV1a"},
+            {
+                "pit": {"id": "46ToAwMDaWR5BXV1a", "keep_alive": "1m"},
+                "query": {"match_all": {}},
+                "sort": [{"timestamp": {"order": "desc"}}],
+                "track_total_hits": False,
+                "search_after": ["1686557542970", "0"],
+            },
+        ),
+    ],
+)
+def test_backends_lrs_es_lrs_backend_query_statements_query(
+    params, expected_body, es_lrs_backend, monkeypatch
+):
+    """Tests the `ESLRSBackend.query_statements` method, given valid statement
```

Comment:
```suggestion
    """Test the `ESLRSBackend.query_statements` method, given valid statement
```

And so on for the other functions of this file :)

Reply:
Well spotted) Thanks) updated the docstrings)